packages/metadata/src/__tests__/unit/decorator-factory.unit.ts

```diff
@@ -116,9 +122,18 @@ describe('ClassDecoratorFactory', () => {
       // eslint-disable-next-line @typescript-eslint/no-unused-vars
       class MyController {}
     }).to.throw(
-      /Decorator cannot be applied more than once on class MyController/,
+      /ClassDecorator cannot be applied more than once on class MyController/,
```

Comment:
+1 to provide a meaningful default name when no `decoratorName` metadata was supplied.